### ISSUE TITLE
`no-unused-disable` does not work within Markdown files

### ISSUE THREAD

**Issue by ehmicky** [mysticatea/eslint-plugin-eslint-comments#23]
`README.md`:

````md
<!-- eslint-disable no-inline-comments -->

```js
var aa = true
```
````

`.eslintrc.yml`:

```yml
plugins:
  - eslint-comments
  - markdown
rules:
  no-inline-comments: 2
  eslint-comments/no-unused-disable: 2
```

In the terminal:

```bash
$ eslint README.md
```
The command does not report any errors. However `no-unused-disable` should be reported.

Versions:
  - `eslint`: `5.16.0` (also fails with `6.0.0-alpha.0`)
  - `eslint-plugin-eslint-comments`: `3.1.1`
  - `eslint-plugin-markdown`: `1.0.0`

I am not sure whether the issue comes from `eslint-plugin-eslint-comments` or `eslint-plugin-markdown`, so I decided to report this bug here.

**Comment by mysticatea** [comment]
Thank you for this report.

`eslint-comments/no-unused-disable` is the alias of [`--report-unused-disable-directives`](https://eslint.org/docs/user-guide/command-line-interface#--report-unused-disable-directives) option.

The markdown plugin seems to handle the HTML comments in a different way from ESLint core, so the rule doesn't recognize the HTML comments. So this is not a bug, but an enhancement.

**Comment by ehmicky** [comment]
Thanks for your answer, you're right. Now I know `eslint-plugin-markdown` is the problem, not `eslint-plugin-eslint-comments`. I've added [an issue](https://github.com/eslint/eslint-plugin-markdown/issues/115) to their repository.
